aip/general/0231.md

```diff
@@ -52,11 +52,19 @@ message BatchGetBooksRequest {
   // Format: publishers/{publisher}
   // If this is set, the parent of all of the books specified in `names`
   // must match this field.
-  string parent = 1;
+  string parent = 1 [
+    (google.api.resource_reference) = {
+      type: "library-example.googleapis.com/Publisher"
```

Comment:
Let's use `child_type` for `parent` fields. And I think I have been using `library` rather than `library_example`.

Reply:
I'll fix the example, but could you elaborate on `child_type` vs `type`? I'm a bit confused by https://google.aip.dev/122#fields-representing-a-resources-parent.